evap/rewards/templates/rewards_index.html

```diff
@@ -22,6 +22,8 @@
                     <form id="reward-redemption-form" action="#" method="POST" class="form-horizontal multiselect-form">
                         {% csrf_token %}
 
+                        <input type="hidden" name="previous_reward_points" value="{{ total_points_available }}">
+                        <input type="hidden" name="previous_redeemed_points" value="{{ total_points_spent }}">
```

Comment:
Good point. However, this would still allow to redeem more points than points that are shown, when reward points are received after page was loaded, which seems rather odd. We could require the previous available points only to be not greater, or do you this edgecase is not relevant?

Reply:
I think this isn't a problem, since users can just redeem more points afterwards if they feel like it.

Reply:
I removed the redeemed reward points field now, but reading your answer once again, I think you might have misunderstood me there:
- user has one reward point and loads the redemption page
- user gains 3 reward points (not shown in UI)
- user (mistakenly) types in 4 points and submits the form
- all server-side checks pass, 4 points are redeemed sucessfully
=> the user might have not anticipated that

Reply:
The user shouldn't be able to enter 4 points in the input / dropdown field, because it should be limited to the available amount of points.

-> For thus, users would have to hand-craft requests -> not a situation where we need to protect users.